### DISCUSSION TITLE
Can't compile for Arduino 101

### DISCUSSION THREAD

**Discussion by amatii** [feilipu/Arduino_FreeRTOS_Library#23] [Q&A]
Hi
while compiling, I got error related to avr/io.h.
==============================================
Arduino: 1.8.3 (Windows 8.1), Board: "Arduino/Genuino 101"

In file included from .../Arduino_FreeRTOS.h:104:0,

from ....\freertos.ino:1:

....\libraries\FreeRTOS\src/FreeRTOSVariant.h:12:20: fatal error: avr/io.h: No such file or directory

 #include <avr/io.h>

                    ^

compilation terminated.

exit status 1
Error compiling for board Arduino/Genuino 101.

**Comment by feilipu** [comment]
Two things:
`Board: "Arduino/Genuino 101"`
and
`#include <avr/io.h>`

This library is only suitable for original AVR based Arduino boards,
as listed on the [README under Compatibility](https://github.com/feilipu/Arduino_FreeRTOS_Library#compatibility).

The "Genuino 101" is an Intel board, using the Curie module.
It isn't supported by this library. And even [Intel has dropped support](https://communities.intel.com/external-link.jspa?url=http%3A%2F%2Fqdms.intel.com%2Fdm%2Fi.aspx%2FC84DF553-5BEC-4813-B415-FF18FF6017E0%2FPCN115582-00.pdf) for it. More detail [here](https://www.theregister.co.uk/2017/07/25/intel_arduino_board_curie_canceled/).

Sorry.
